.gitignore

```diff
@@ -1,3 +1,157 @@
-node_modules/
-dist/
+### WebStorm ###
```

Comment:
Sorry for this brutal change. I pulled it from [gitignore.io](https://www.gitignore.io/) to cover most of the files generated from Webstorm and other IDE's that contributors may use. This gitignore also contains Node's common ingore rules. But I could merge Webstorm and Node specific rules if you wish, tell me.